>>> lazy_matrix_mul = __import__('101-lazy_matrix_mul').lazy_matrix_mul

>>> print(lazy_matrix_mul([[45, 10], [32, 41]], [[4, 9], [1, 3]]))
[[190 435]
 [169 411]]
>>> print(lazy_matrix_mul([[-4.0, -35.0]], [[-90], [-12.0]]))
[[780.]]
>>> print(lazy_matrix_mul("Holberton", [[4, 9], [1, 3]]))
Traceback (most recent call last):
ValueError: Scalar operands are not allowed, use '*' instead
>>> print(lazy_matrix_mul([[4, 9], [1, 3]], "Holberton"))
Traceback (most recent call last):
ValueError: Scalar operands are not allowed, use '*' instead
>>> print(lazy_matrix_mul([[]], [[5, 6], [7, 8]]))
Traceback (most recent call last):
ValueError: shapes (1,0) and (2,2) not aligned: 0 (dim 1) != 2 (dim 0)
>>> print(lazy_matrix_mul([[5, 6], [7, 8]], [[]]))
Traceback (most recent call last):
ValueError: shapes (2,2) and (1,0) not aligned: 2 (dim 1) != 1 (dim 0)
>>> print(lazy_matrix_mul([[5, "6"], [7, 8]], [[5, 6], [7, 8]]))
Traceback (most recent call last):
TypeError: invalid data type for einsum
>>> print(lazy_matrix_mul([[5, 6], [7, 8]], [[5, "6"], [7, 8]]))
Traceback (most recent call last):
TypeError: invalid data type for einsum
>>> print(lazy_matrix_mul([[5, 6, 10], [7, 8]], [[5, 6], [7, 8]]))
Traceback (most recent call last):
ValueError: setting an array element with a sequence.
>>> print(lazy_matrix_mul([[5, 6], [7, 8]], [[5, 6, 1], [7, 8]]))
Traceback (most recent call last):
ValueError: setting an array element with a sequence.
>>> print(lazy_matrix_mul([[1, 2, 3], [3, 4, 5]], [[1, 2], [3, 4]]))
Traceback (most recent call last):
ValueError: shapes (2,3) and (2,2) not aligned: 3 (dim 1) != 2 (dim 0)
>>> print(lazy_matrix_mul([[1, 2, 3], [3, 4, 5]]))
Traceback (most recent call last):
TypeError: lazy_matrix_mul() missing 1 required positional argument: 'm_b'
>>> print(lazy_matrix_mul())
Traceback (most recent call last):
TypeError: lazy_matrix_mul() missing 2 required positional arguments: 'm_a' and 'm_b'